src/Arduino_CRC16.h

```diff
@@ -0,0 +1,12 @@
+/*
+    This file is part of the Arduino_CloudUtils library.
+
+    Copyright (c) 2024 Arduino SA
+
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+#pragma once
+
+#include "./crc/crc16.h"
```

Comment:
Add an empty line at the end of the file 

```suggestion
#include "./crc/crc16.h"

```